### PR TITLE
Add information about the E1746

### PR DESCRIPTION
**Pull request by realPy**
This information can be useful for people want track the availability of this device

### CODE REVIEW

File: docs/devices/E1746.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ While pairing the LED is flashing/dimming slowly. Once the pairing is finished,
 ## OTA updates
 This device supports OTA updates, for more information see [OTA updates](../information/ota_updates.md).
 
+## Keep alive payload
+The device send a payload exactly 1 hours after the first pairing and a payload each exactly 4 hours after.
```
Comment (Koenkk) [inline docs/devices/E1746.md:30]: By just reading this text, it's unclear to me what todo/expect from it.

Comment (realPy) [reply docs/devices/E1746.md:30]: This device send one empty payload each 4 hours. If your last seen is more than 4 hours, so the device stop to work.
It's very usefull to monitoring devices that stop to work.

Comment (Koenkk) [reply docs/devices/E1746.md:30]: Then please add this info

**Comment by realPy** [comment]
Info this complete

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**